flake-module.nix

```
@@ -51,6 +51,19 @@ in
               };
             };
           };
+          packageAttrsSubmodule = with types; submodule {
+            options = {
+              root = mkOption {
+                type = path;
+                description = "Path to Haskell package where the .cabal file lives";
+              };
+              modifier = mkOption {
+                type = functionTo package;
+                default = drv: drv;
+                description = "Modifier to be applied to Haskell package before applying to overlay";
```

```suggestion
                description = "Modifier to be applied to Haskell package before adding to overlay";
```

Avoids association with function application.

---

LGTM